### PR TITLE
Add logs from elasticsearch to app detail page

### PR DESCRIPTION
**Pull request by andyhd**
Shows the app logs from the last 24 hours

### CODE REVIEW

File: controlpanel/frontend/static/components/app-logs/macro.html [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+  </div>
+  <p class="govuk-body">
+    <a href="{{ kibana_link(app) }}">View logs in Kibana</a>
+  </p>
+{% endmacro %}
+
+{% macro kibana_link(app) -%}
 {% filter replace(" ", "") -%}
 {% filter replace("\n", "") -%}
 https://kibana.services.{{env}}.mojanalytics.xyz/app/kibana#/discover?_g=(
```
Comment (r4vi) [inline controlpanel/frontend/static/components/app-logs/macro.html]: maybe we should start parameterising the domain now because it's going to change soon

Comment (andyhd) [reply controlpanel/frontend/static/components/app-logs/macro.html]: Good point

File: controlpanel/api/elasticsearch.py [hunk 2]
```diff
@@ -23,3 +24,32 @@ def bucket_hits_aggregation(bucket_name, num_days=None):
                   size=100)
 
     return s.execute().aggregations.bucket_hits
+
+
+def app_logs(app, num_hours=None):
+    if not num_hours:
```
Comment (xoen) [inline controlpanel/api/elasticsearch.py]: maybe I'm missing something, I'm a Python noon after all, but what's the difference between this and having the `num_hours` parameter have a default of `1`: `def app_logs(app, num_hours=1):`?

File: controlpanel/frontend/static/components/app-logs/macro.html [hunk 3]
```diff
@@ -1,7 +1,18 @@
-{% macro app_logs(app) -%}
+{% macro app_logs(app, kibana_base_url) %}
+  <div class="app-logs">
+    <pre>{% for entry in app.get_logs() -%}
```
Comment (xoen) [inline controlpanel/frontend/static/components/app-logs/macro.html]: nice one :)

File: controlpanel/api/elasticsearch.py (hunk 2 above)
Comment (andyhd) [reply controlpanel/api/elasticsearch.py]: Yep, dunno how I ended up writing this

**Pull request merged**